nixio/pycore/h5dataset.py

```diff
@@ -43,6 +43,8 @@ def write_data(self, data, count=None, offset=None):
             self.dataset[:] = data
 
     def read_data(self, data, count=None, offset=None):
+        if 0 in self.dataset.shape or len(self.dataset.shape) == 0:
+            return
```

Comment:
in matlab something like ```(2, 0, 1)``` would be just fine, In fact it has only 2 dimensions and if you call squeeze on it, it would leave a 2D matrix. Not sure whether same things happen with numpy. I agree, from the mathematical point of view this matrix would be empty.